website/docs/components/application-state/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,25 @@
+## Usage
+
+### When to use
+- When a user's action does not yield any results due to incorrect or incomplete input.
+- When an application encounters an issue or error during its operation.
+
+### When not to use
+- When the absence of content is expected and does not require an explanation to the user. 
+- When there is a clear and intuitive way to add or populate content. 
+
+## Icon
+
+- The icon should be relevant and communicate the same purpose as the errorCode.
+
+## Actions
+
+- In the footer, you can include up to two stand-alone links. 
+- It’s important to keep these links separate from call-to-action buttons (CTA), as doing so may cause misalignment with the code.
+- For the standalone link, we recommend using the default medium size but use whichever size fits best in the UI.
```

Comment:
@andgen404 Since the rest of the page is likely to be pretty empty, and we provide them with the styling for the rest of the content within the ApplicationState component, we're already determining the best size for the UI.

```suggestion
- For the standalone link, we recommend using the medium size.
```